clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -122,6 +123,48 @@ public void setup() {
         this.props.put(VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
     }
 
+    @Test
+    public void testOffsetFetchRequestStateToStringBase() {
+        ConsumerConfig config = mock(ConsumerConfig.class);
+        CommitRequestManager.MemberInfo memberInfo = new CommitRequestManager.MemberInfo();
```

Comment:
As we had discussion about "Optional#toString", maybe we define either `MemberInfo#memberId` or `MemberInfo#memberEpoch`, and then we should check the `MemberInfo#toString` does not contain "Optional[value]"

Reply:
I see what you meant now, I have implemented a setter for ```MemberInfo#memberEpoch``` and set the memberEpoch value. Upon testing, there is no "Optional[value]"

Reply:
@brenden20 we already have an `onMemberEpochUpdated` that you could use for this, so we don't have to add a new setter. That's the actual method that gets called when the member gets a new epoch on the heartbeat response, and we want to pass it on to the commit manager ;)

Reply:
@lianetm oh I see it now, I will revert that change and use ```onMemberEpochUpdated``` instead. Thank you!